circles-app/src/lib/stores/wallet.svelte.ts

```
@@ -14,123 +14,116 @@ import { getCirclesConfig } from '$lib/utils/helpers';
-      await getCirclesConfig(100n, environment.ring),
+      runner,
+      await getCirclesConfig(100n, rings),
     );
     circles.set(sdk);
+    let savedGroup = CirclesStorage.getInstance().group;
+
 
-    if (avatarState.isGroup && savedGroup) {
-      avatarState.groupType = await sdk.getGroupType(savedGroup);
+    if (savedGroup) {
+      let groupType = CirclesStorage.getInstance().groupType;
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'groupType' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)